torch/_dynamo/variables/builder.py

```diff
@@ -3956,7 +4125,7 @@ def make_type_handlers():
             )
         )
 
-        def passthrough(tx: "InstructionTranslator", value):
+        def passthrough(tx: "InstructionTranslator", value: Any) -> Any:
```

Comment:
```suggestion
        def passthrough(tx: "InstructionTranslator", value: T) -> T:
```
No need to type erasure with a Pass Through func!